Makefile

```diff
@@ -204,3 +204,44 @@ migrate: migrate-lms migrate-cms
 # Part of https://github.com/openedx/wg-developer-experience/issues/136
 ubuntu-requirements: ## Install ubuntu 22.04 system packages needed for `pip install` to work on ubuntu.
 	sudo apt install libmysqlclient-dev libxmlsec1-dev
+
+eslint:	## check javascript for quality issues
+	python scripts/eslint.py
+
+xsslint: ## check xss for quality issuest
+	python scripts/xsslint/xss_linter.py \
+	--rule-totals \
+	--config=scripts.xsslint_config \
+	--thresholds=scripts/xsslint_thresholds.json
+
+pycodestyle: ## check python files for quality issues 
+	pycodestyle .
+
+## Re-enable --lint flag when this issue https://github.com/openedx/edx-platform/issues/35775 is resolved
+pii_check: ## check django models for pii annotations
+	DJANGO_SETTINGS_MODULE=cms.envs.test \
+	code_annotations django_find_annotations \
+		--config_file .pii_annotations.yml \
+		--app_name cms \
+		--coverage
+	
+	DJANGO_SETTINGS_MODULE=lms.envs.test \
+	code_annotations django_find_annotations \
+		--config_file .pii_annotations.yml \
+		--app_name lms \
+		--coverage		
```

Comment:
Could you make the following changes, which would allow you to put `--lint` back in this PR?

* upgrade the edx-proctoring pin to from 4.18.3 to 4.18.4
* remove celery_utils.FailedTask from the .annotation_safe_list.yml

Reply:
@kdmccormick Even after doing the mentioned steps, upgrade edx-proctoring and remove the celery_utils.FailedTask, when i use --lint flag it starts showing linting errors. 
https://github.com/openedx/edx-platform/actions/runs/12124064673/job/33800928426?pr=35159